sdks/unreal/Agones/Source/Agones/Private/AgonesComponent.cpp

```diff
@@ -70,11 +77,20 @@ void UAgonesComponent::HealthPing(const float RateSeconds)
 	}
 
 	FTimerDelegate TimerDel;
-	TimerDel.BindUObject(this, &UAgonesComponent::Health, FHealthDelegate(), FAgonesErrorDelegate());
+	TimerDel.BindUObject(this, &UAgonesComponent::HealthCheck);

```

Comment:
If it were to refactor this so that a delegate is passed to the WatchGameServer this would be done inside there instead 👍 IMO it would make it much cleaner and more epxlicit